msteams-platform/whats-new.md

```diff
@@ -519,6 +519,8 @@ Discover Microsoft Teams platform features that are deprecated. You can now get
 
 Teams platform features that aren't available.
 
+* ***October 30, 2024***: Top-level domain third-party cookies in iframes are blocked, with deprecation rolling out on Chrome (1%) and soon Firefox and Safari.
```

Comment:
Also here--let's generalize the description ("across all major browsers") and avoid reporting on rollout status. Better yet, rephrase this as a CTA, e.g., "Future-proof your app for third-party cookie deprecation across major browsers"

Reply:
UPdated as suggested.